pkg/ansible/generate_test.go

```diff
@@ -99,12 +99,14 @@ func TestGenerateClusterVars(t *testing.T) {
 	tests := []struct {
 		name             string
 		cluster          *coapi.Cluster
+		clusterVersion   *coapi.ClusterVersion
 		shouldInclude    []string
 		shouldNotInclude []string
 	}{
 		{
 			name:    "cluster",
 			cluster: testCluster(),
+			clusterVersion: testClusterVersion(),
```

Comment:
Just need to update these tests with additional string to look for the deployment type you're passing in. (shouldInclude array)